examples/qwp/pool/main.go

```
@@ -0,0 +1,86 @@
+	qdb "github.com/questdb/go-questdb-client/v4"
+)
+
+func main() {
+	ctx := context.TODO()
+
+	// One cluster config for both directions. List every node in one addr.
+	db, err := qdb.Connect(ctx, "ws::addr=localhost:9000;")
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer db.Close(ctx)
```

golangci-lint (2.12.2)

[error] Error return value of `db.Close` is not checked

(errcheck)